code/modules/mob/living/blood.dm

```diff
@@ -29,11 +29,11 @@
 			if(!heart)
```

Comment:
FYI you need to remove this part of the code or heart effects aren't removed. How heart damage works is by reducing "effective" blood level. For example, if you've got 100% blood and a bruised heart, you'll get the effects of 70% blood. If you've got 90% blood and a broken heart you'll get the effects of 45% blood. This also drains blood

I don't know if you've axed it in another part of the code or not, but this was happening to me in-game during the TM so I'm pretty sure it's not removed